docker/apps/agreement_database/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM vickumar/python-nodejs
+
+RUN git clone https://github.cfpb.gov/CFPB/agreement_database.git /apps/agreement_database/
```

Comment:
I can put the URLs as environment settings in the base image, perhaps.

So it would read something like:

```
RUN git clone $AGREEMENT_DB_GIT_URL /apps/agreement_database/
```

and then do the same thing for the other apps.